src/phylum/init/sig.py

```diff
@@ -1,118 +1,73 @@
-"""Helper functions for verifying minisign signatures.
+"""Verify Phylum generated digital signatures.
 
-This module is meant to be a quick and dirty means of verifying minisign signatures in Python.
-There is no readily accessible Python library at this time. The `py-minisign` repository exists
-on GitHub to attempt this - https://github.com/x13a/py-minisign - but it does not exist as a
-package on PyPI and also does not appear to be actively maintained. There is a `minisign` package
-on PyPI - https://pypi.org/project/minisign/ - but it comes from a different repo and has no
-functionality at the time of this writing.
+This module is meant to be a quick and dirty means of verifying RSA signatures in Python.
+It makes use of the hazardous materials layer of the `cryptography` library, but does so
+in a way that closely follows the example documentation:
```

Comment:
Honestly I wouldn't have much faith in security when reading "quick and dirty means of verifying RSA signatures in Python". I feel like "simple" probably expresses the goal of the module more appropriately without inducing security doubts?

Reply:
Agreed. That language was a holdover from the minisign logic, which was implemented with a bit more granularity since there were no existing libraries that could be used to do the job. This language will be updated.